### ISSUE TITLE
Junos rfc-compliant mode not working

### ISSUE THREAD

**Issue by door7302** [openshift-telco/go-netconf-client#5]
Hello, 

On Junos equipment when you configure netconf as follows:

regress@rtme-mx-25> show configuration system services netconf 
ssh {
    port 830;
}
rfc-compliant;   <<<< Option 
}

Junos replies with  all NETCONF tags including the nc prefix. Sample reply when this option is set: £`

```xml
<nc:rpc-reply
	xmlns:nc="urn:ietf:params:xml:ns:netconf:base:1.0"
	xmlns:junos="http://xml.juniper.net/junos/22.2I0/junos">
	<interface-information
		xmlns="http://xml.juniper.net/junos/22.2I0/junos-interface" junos:style="description">
		<physical-interface>
			<name>et-0/1/5</name>
			<admin-status>up</admin-status>
			<oper-status>up</oper-status>
			<description>TEST_Description</description>
		</physical-interface>
	</interface-information></nc:rpc-reply>
]]>]]>
```
It seems your module doesn't accept rpc-reply prefixed with "nc". I faced this issue: " timeout while executing request". 

Without the option your module works pretty well with our Juniper devices. My tool relies on your lib and I would like to support both configs with and without the rfc-compliant option. Do you know if a fix may be considered? 

Thank you so much. 
Best Regards
David

More info:  https://www.juniper.net/documentation/us/en/software/junos/netconf/topics/concept/netconf-session-rfc-compliant.html

**Comment by door7302** [comment]
Hello, Any news ?

**Comment by adetalhouet** [comment]
@door7302 just submitted a patch in the main branch. Can you check with the latest code if things work as expected?

Thanks

**Comment by door7302** [comment]
Hello, 

I've just tested... It works... thank you so much.

David

**Comment by adetalhouet** [comment]
I've released a new version, v1.0.6, containing the fix.
